docs/develop/go/message-passing.mdx

```diff
@@ -98,64 +131,142 @@ func YourWorkflowDefinition(ctx workflow.Context, param YourWorkflowParam) error
  			selector.Select(ctx)
  		}
  	})
-   // submit activity one
-   // signal can be received while activity one is pending
-
+   // You could now submit an activity; any signals will still be received while the activity is pending.
  }
```

Comment:
@Quinn-With-Two-Ns I reinstated the section on processing signals in a `workflow.Go` routine. It had been deleted in the diff between this branch and `main`.